include/realizations/catchment/Bmi_Module_Formulation.hpp

```diff
@@ -44,18 +37,12 @@ namespace realization {
          * @param output_stream
          */
         Bmi_Module_Formulation(std::string id, std::shared_ptr<data_access::GenericDataProvider> forcing_provider, utils::StreamHandler output_stream)
-                : Bmi_Formulation(std::move(id), forcing_provider, output_stream) { };
+                : Bmi_Formulation(std::move(id), forcing_provider, output_stream) { }
 
-        virtual ~Bmi_Module_Formulation() {};
+        virtual ~Bmi_Module_Formulation() = default;
```

Comment:
Not an issue, but since we're modifying this line anyways, could we change the syntax to use `override` instead of `virtual` to show that the virtual destructor hierarchy starts above this class?

Reply:
I was kinda tempted to just leave off `virtual` too. I think technically, it's not an `override`, since the parent class destructor is still called as part of object instance destruction. Syntactically, it may be allowed, though.

Reply:
Ok, yeah, I see `override` is allowed as a special case. I'll make the change to add it.